runtimes/pythonrt/runner_manager.go

```diff
@@ -54,16 +64,17 @@ func waitForServer(name string, h Healther, timeout time.Duration) error {
 	return fmt.Errorf("%s not ready after %v", name, timeout)
 }
 
-func dialRunner(addr string) (pb.RunnerClient, error) {
+func dialRunner(addr string) (*RunnerClient, error) {
 	creds := insecure.NewCredentials()
 	conn, err := grpc.NewClient(addr, grpc.WithTransportCredentials(creds))
 	if err != nil {
 		return nil, err
 	}
 
-	c := pb.NewRunnerClient(conn)
+	c := &RunnerClient{pb.NewRunnerClient(conn), conn}
```

Comment:
I prefer to use pointer only when returning, this way it clear on return that we use pointer semantics.

```
c := RunnerClient{pb.NewRunnerClient(conn), conn}
...

return &c, nil
```

Reply:
done